htmlmaker_preprocessing.ps1

```diff
@@ -16,6 +18,7 @@ $currVol=split-path $currVolPath -Qualifier	#C: or S:
 $filenameSplit=split-path $inputFile -Leaf			#file name without path
 Write-Host "Input file is $filenameSplit"
 $filename=$filenameSplit.SubString(0, $filenameSplit.LastIndexOf('.')).replace(' ','')	#filename w/out extension or spaces
+$fileType=$filenameSplit.SubString(1, $filenameSplit.LastIndexOf('.'))  # just extension
```

Comment:
Moved from line 35. Needs to be actual file extension, so that we can process every file not just .doc files.